mapgen/mapgen.cpp

```diff
@@ -71,7 +71,7 @@ void generateMap(int count, vector<Coord>& stores, vector<Coord>& users, vector<
 	copyFile(part, map);
 	part.close();
 
-	map<<newStores[count]<<","<<endl;
+	map<<newStores[0]<<","<<endl;
```

Comment:
Nu era suprascris `vector<<` asa ca printa `(0,0)` mereu, count era 1. Dar in html nu era tratat cazul pt mai multe magazine, asa ca n-am tratat nici aici.